testutil/integration/example_test.go

```diff
@@ -29,6 +32,12 @@ func Example() {
 	keys := storetypes.NewKVStoreKeys(authtypes.StoreKey, minttypes.StoreKey)
 	authority := authtypes.NewModuleAddress("gov").String()
 
+	// replace the logger by testing values in a real test case (e.g. log.NewTestLogger(t))
+	logger := log.NewLogger(io.Discard, log.OutputJSONOption())
```

Comment:
Why are we discarding log output, yet using JSON? Surely there must be a shorthand for just getting a no-op logger.